### PR TITLE
Configurable targeting prefix

### PR DESCRIPTION
**Pull request by KolomiietsSerhii**
Related to:
https://github.com/prebid/prebid-server/issues/3094

### CODE REVIEW

File: src/main/java/org/prebid/server/auction/TargetingKeywordsCreator.java [hunk 1]
```diff
@@ -31,54 +39,60 @@ public class TargetingKeywordsCreator {
      * Exists to support the Prebid Universal Creative. If it exists, the only legal value is mobile-app.
      * It will exist only if the incoming bidRequest defiend request.app instead of request.site.
      */
-    private static final String HB_ENV_KEY = "hb_env";
+    private static final String ENV_KEY = "env";
     /**
      * Used as a value for HB_ENV_KEY.
```
Comment (SerhiiNahornyi) [inline src/main/java/org/prebid/server/auction/TargetingKeywordsCreator.java:44]: Comment should be updated

File: src/test/java/org/prebid/server/auction/TargetingKeywordsCreatorTest.java [hunk 2]
```diff
@@ -520,10 +563,86 @@ public void shouldNotIncludeDealBidTargetingIfAlwaysIncludeDealsFlagIsFalse() {
...
                 .makeFor(bid, "bidder1", false, null, null, null, null);
 
         // then
         assertThat(keywords).doesNotContainKeys("hb_bidder_bidder1", "hb_deal_bidder1", "hb_pb_bidder1");
     }
+
+    @Test
+    public void shouldReturnTargetingKeywordsWithCustomPrefixFromAccountForOrdinaryBidOpenrtb() {
+        // given
+        final String expected = "test_";
```
Comment (SerhiiNahornyi) [inline src/test/java/org/prebid/server/auction/TargetingKeywordsCreatorTest.java:578]: customPrefix will be better name

**Review by SerhiiNahornyi — CHANGES_REQUESTED** [review]

File: src/main/java/org/prebid/server/auction/TargetingKeywordsCreator.java (hunk 1 above)
Comment (KolomiietsSerhii) [reply src/main/java/org/prebid/server/auction/TargetingKeywordsCreator.java]: Done.

File: src/test/java/org/prebid/server/auction/TargetingKeywordsCreatorTest.java (hunk 2 above)
Comment (KolomiietsSerhii) [reply src/test/java/org/prebid/server/auction/TargetingKeywordsCreatorTest.java]: Done.

**Review by SerhiiNahornyi — APPROVED** [review]

**Pull request merged**